test/acceptance/behave/components/ngsiv2/entities/list_entities/list_entities.feature

```diff
@@ -0,0 +1,808 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed:
+#        -t=-skip
+#        the "too_slow" tag is used to mark scenarios that running are too slow, if would you like to skip these scenarios:
+#        -t=-too_slow
+
+
+Feature: list all entities with get request and queries parameters using NGSI v2. "GET" - /v2/entities/
+  Queries parameters
+     tested : limit, offset, id, idPattern, type, q and option=count,keyValues
+     pending: georel, geometry, coords and option=values
```

Comment:
If you want to be comprehensive, you could also add option=distinct (#1050) to the pending list.